### PR TITLE
Update to Leap 15.6 where applicable

### PR DESCRIPTION
**Pull request by okurz**
After
* #5629

Related progress issue: https://progress.opensuse.org/issues/157984

### CODE REVIEW

**Comment by okurz** [comment]
@mergifyio rebase

File: container/webui/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: MIT
 #!BuildTag: openqa_webui:latest opensuse/openqa-webui:latest opensuse/openqa-webui:%PKG_VERSION% opensuse/openqa-webui:%PKG_VERSION%.%RELEASE%
-FROM opensuse/leap:15.5
+FROM opensuse/leap:15.6
```
Comment (baierjan) [inline container/webui/Dockerfile:3]: See https://github.com/os-autoinst/openQA/actions/runs/9008630845/job/24768938320#step:3:4155

According to `zypper info --requires apache2` apache2-utils (and hence `gensslcert`) is no longer a required package for apache2 in Leap 15.6.

I suggest a small change on line 21 in this Dockerfile

```diff
- zypper in -y ca-certificates-mozilla curl openQA-local-db apache2 hostname which w3m 'rubygem(sass)' && \
+ zypper in -y ca-certificates-mozilla curl openQA-local-db apache2 apache2-utils hostname which w3m 'rubygem(sass)' && \
```

**Review by baierjan — CHANGES_REQUESTED** [review]

File: container/webui/Dockerfile (hunk 1 above)
Comment (okurz) [reply container/webui/Dockerfile:3]: sure, done

**Comment by okurz** [comment]
To address the circle CI dependency issues as encountered in https://app.circleci.com/pipelines/github/os-autoinst/openQA/13502/workflows/935d6c71-1b1f-426d-9725-12afc6ecd6eb/jobs/126668?invite=true#step-108-76

Package 'perl-Devel-Cover-1.40' not found.
Package 'perl-Test-MockModule-0.177.0' not found.

applying same as in [#157996-7](https://progress.opensuse.org/issues/157996#note-7)

for i in perl-Devel-Cover perl-Test-MockModule; do osc linkpac devel:openQA:Leap:15.5 $i devel:openQA:Leap:15.6; done

and according comments on the packages

**Comment by okurz** [comment]
Needed to enable 15.6 in https://build.opensuse.org/project/show/devel:openQA:ci project config

**Comment by okurz** [comment]
I triggered the "dependencies-PR" run to have updated dependencies:
* #5632

**Review by baierjan — APPROVED** [review]

**Review by perlpunk — APPROVED** [review]

**Pull request merged**